packages/core/src/indexing/local-index-api.ts

```diff
@@ -43,6 +43,10 @@ export class LocalIndexApi implements IndexApi {
     await this.databaseIndexApi.indexStream(args)
   }
 
+  async queryCount(query: BaseQuery): Promise<number> {
```

Comment:
Do you expect it to return number of _all_ relevant records?

Reply:
Yes all the records matching the query parameters (account, model and filter(s)).